server/controllers/getGamesList.js

```diff
@@ -13,12 +13,17 @@ export async function createGamesList(req, res) {
         [headers.BggFilterMaxDuration]: req.query.maxDuration,
         [headers.BggFieldWhitelist]: headers.defaultWhitelist
     };
-    const gamesList = await makeRequest(requestUrl, headers.filterHeaders(requestHeaders));
-    res.json(getRandomGame(gamesList));
+    return await makeRequest(requestUrl, headers.filterHeaders(requestHeaders));
```

Comment:
`return await` is unnecessary and a bit weird.  If `makeRequest` that promise is what will be returned either way.

You may wish to leave the `async` keyword here though, just to signal that this function does return a promise